documentation/man/features/mail.rst

```diff
@@ -90,3 +103,7 @@ variables at once::
 To send the last commit as a patch just write::
 
   kw mail --send --to=some@email.com
+
+To send a range of commits the following can be used::
+
+  kw mail -s <SHA1>..<SHA2>
```

Comment:
This commit adds the ability to specify the range of commits, hence this being here. I could make a dedicated commit before this, but I did it this way because I use the pass through implementation to give these commit references to `git send-email`